src/styles/blocks/card.scss

```diff
@@ -0,0 +1,131 @@
+$w-cont: #fff;
+$color-b: #060b35;
+$cont-color: #616070;
+$blue-cont: #00acdc;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 200px;
+  border-radius: 5px;
+  background-color: #{$w-cont};
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 0 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #{$color-b};
+    display: flex;
+    margin: 0;
+    width: 166px;
+    word-break: break-word;
+    margin-bottom: 4px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #{$cont-color};
+    display: flex;
+    margin: 0;
+  }
+
+  &__images {
+    display: flex;
+    height: 134px;
+    padding-right: 3px;
+    padding-left: 3px;
+  }
+
+  &__global {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    height: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    color: #{$color-b};
+    margin: 0;
+    align-items: center;
+    display: flex;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #{$cont-color};
+  }
+
+  &__value {
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #{$w-cont};
+    border-radius: 5px;
+    background-color: #{$blue-cont};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for element with content inside. Just use padding: 11px 0;
```suggestion
```